celery.yml

```diff
@@ -19,8 +19,8 @@
         enable_create_user: true
         enable_kernel_5: true
         enable_install_software: true
+    - usegalaxy_eu.galaxy_systemd
```

Comment:
keep in mind that the order of playbooks might be relevant ... so maybe you need to change the comments ...